source/includes/api/_user_management.md

```diff
@@ -155,35 +249,40 @@ curl -X POST http://api.resourcewatch.org/auth/sign-up?origin=rw \
 }'
 ```
 
-#### Permissions
+Account creation endpoint, for accounts using email + password based login for both HTML and JSON requests.
 
-Based on roles, different types of users can create new users with different roles:
+The combination of both `user email` and `provider` must be unique - a given email address may be associated with multiple, non-related user accounts by using different authentication providers (email+password, facebook, twitter, etc).
 
-- ADMIN: Can create any type of user
-- MANAGER: Can create a user of type `MANAGER` or `USER`.
-- Public users: Can register themselves in the API, being assigned the `USER` role.
+For HTML requests, it will display a message informing about any validation error, or informing the user in case of success.
 
+For JSON requests, successful logins will return a JSON object containing the details of the user.
 
-### Confirm user account
+Keep in mind that this endpoint creates a **deactivated** user account. A successful call to this endpoint send an email to the user, with a link that the user must click in order to confirm their account. Once confirmed using this process, the user account becomes activated and fully functional, and the user will be able to log in.
 
-Endpoint used in the user validation email to confirm the address upon registration.
+The email sent to the user will have the identity of the `origin` app provided on the request, with a system-wide fallback (GFW) being used in case none is provided.
 
-It accepts an optional `callbackUrl` query parameter with an URL to which the user will be redirect if the confirmation succeeds.
+While optional, it's highly recommended that you specify which apps the user will be granted access to, as most API operation validate the user's apps match datasets, widgets, etc. All accounts created this way will have the `USER` role.
 
-Should no `callbackUrl` be provided, the user is redirected to an URL based on the first application associated to their user account - see `ct-oauth-plugin` configuration for more info.
+**Errors**
+
+Error code     | Error message  | Description
+-------------- | -------------- | --------------
+422            | Email exists.  | An account already exists for the provided email address.
+422            | Email, Password and Repeat password are required.  | You are missing one of the required fields.
```

Comment:
Same code as the previous one?

Reply:
You mean the HTTP error code for both scenarios? yes, 422 would fit both scenarios, IMO 
https://www.keycdn.com/support/422-unprocessable-entity